src/index.ts

```diff
@@ -1,16 +1,19 @@
-// Copyright 2016 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
```

Comment:
Some time ago we tried to standardize on `//` style comments in our modules instead of `/** **/` for license headers.  To use `@license`, do you need to use multiline comments?  Or would single line comments continue to work?

Reply:
Hmm just tool a look at the import script in g3 and it appears to only look for `@license` in multiline comments.

If this is a deal breaker I can just update the license manually in g3 for now, add a local modification note. I can also file a bug to see about supporting single line comments in the importer.

Let me know which you prefer, but yeah at least for now, this style of license comment doesn't play well with the g3 importer :/

Reply:
So this could be possibly be fine.  We have ~100 node.js GitHub repositories, and we like to keep things consistent.  If you need this added everywhere, it may be faster us to script a broad change that fixes the header in every file all at once 😬   

@alexander-fenster @bcoe this could be an adventure

Reply:
A starting point might be updating the [typescript generator](https://github.com/googleapis/gapic-generator-typescript) and the templates [here](https://github.com/googleapis/synthtool/blob/master/synthtool/languages/node.py) which are also populated in repositories.

@nathanmosher if this change provides enough value to be worth this work, I would start by making changes in these places.

Are there specific libraries in g3 that you'd like the headers for?